src/components/Sidebar.tsx

```diff
@@ -1,13 +1,11 @@
-import React, { useEffect, useState } from "react";
-import { trpc } from "../utils/trpc";
+import React, { useState } from "react";
 import InfiniteScroll from "react-infinite-scroll-component";
 import Spinner from "./Spinner";
-import { Role, Status, User } from "@prisma/client";
 import mapboxgl, { Marker } from "mapbox-gl";
-import { min } from "lodash";
+import { PublicUser } from "../utils/types";
 
 type ScrollableList = {
```

Comment:
this isn't used in the code, can we delete? @devashish9 @DavidOB1

Reply:
I actually don't want you to edit this page, delete any changes to this since Jun worked on this page himself. We can come back to this page after Jun's changes have been merged in and change everything to use the Public User type.